.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: elixir
 
 elixir:
-  - 1.6.0
+  - 1.7.4
```

Comment:
This may be hard without making code changes.  Stuff changed.

Reply:
Ok, noted! Do you think it would need a lot of code changes even with only 1.7.1?